data/legend/metadata/dataprod/config/validity.yaml

```
@@ -0,0 +1,8 @@
+- valid_from: 20221118T000000Z
+  category: cal
+  apply:
+    - L200-p02-r%-T%-all-config.json
+    - L200-p02-r%-T%-cal-config.json
+- valid_from: 20221118T000000Z
+  apply:
+    - L200-p02-r%-T%-all-config.json
```

Our code breaks here:
```suggestion
- valid_from: 20221118T000000Z
  category: cal
  apply:
    - L200-p02-r%-T%-all-config.yaml
    - L200-p02-r%-T%-cal-config.yaml
- valid_from: 20221118T000000Z
  apply:
    - L200-p02-r%-T%-all-config.yaml
```